packages/strapi-design-system/package.json

```diff
@@ -57,15 +59,16 @@
     "react-router-dom": "^5.2.0",
     "styled-components": "^5.2.1"
   },
+  "peerDependenciesMeta": {
+    "react-router-dom": {
+      "optional": true
+    }
+  },
   "scripts": {
-    "analyze:bundle": "cross-env BUNDLE_ANALYZE=1 webpack",
+    "dev": "vite",
     "prebuild": "rimraf dist",
-    "build": "yarn build:files && cross-env IS_V2=true yarn build:files",
-    "build:files": "yarn build:dev && yarn build:prod && yarn copy:files && yarn create:index-files",
-    "build:dev": "cross-env NODE_ENV=development webpack-cli",
-    "build:prod": "cross-env NODE_ENV=production webpack-cli",
-    "copy:files": "node ../../tools/copy-files.js",
-    "create:index-files": "node ./createIndexFiles.js",
+    "build": "yarn build:prod",
+    "build:prod": "vite build",
```

Comment:
Does it make a difference to set `NODE_ENV=production` here? Just to make sure the person who releases the DS publishes the most efficient bundles ...

Reply:
I don't believe so, it by default runs build in `production` mode – https://vitejs.dev/guide/env-and-mode.html#modes